astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -1542,6 +1522,35 @@ def test_spherical_offsets():
     assert_allclose(dra, 0*u.arcmin)
     assert_allclose(ddec, np.arange(4)*u.arcmin)
 
+    # spherical_offsets_by only supports scalar frame data:
```

Comment:
I think this should be its own test function - now it is just part of `spherical_offsets_to` tests. Perhaps easiest to make it a class?

Reply:
I removed the raise, so I think this can just go away